test-renku/README.md

```diff
@@ -0,0 +1,7 @@
+# Run Renku Acceptance tests
+
+This is a composit action that:
+- sets up helm
+- run the helm tests
+- downloads test results from S3
+- optionally cleans up the CI deployment at the end
```

Comment:
could you please add a list/table of inputs?

Reply:
Yeah I will do that.

Reply:
Done.